nodes-media/package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "@desci-labs/nodes-media",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Resolve open access data, transcode media, translate LaTeX to PDF",
-  "engines": {
-    "node": ">=16"
-  },
+  "type": "module",
   "scripts": {
-    "dev": "debug=* NODE_PATH=./src ts-node-dev --inspect=0.0.0.0:9230 --respawn --exit-child ./src/index.ts",
+    "dev": "tsx watch --inspect=0.0.0.0:9230 ./src/index.ts",
     "build": "rimraf dist && tsc",
-    "start": "NODE_PATH=./dist node ./dist/index.js",
```

Comment:
This is unnecessary when the cursed and deprecated `baseUrl` config is removed, as now the paths in the emitted js files make sense without being clairvoyant